### ISSUE TITLE
Could you support snakeCase with uppercase letters

### ISSUE THREAD

**Issue by nobyna** [yuanhjty/code-template-tool#6]
Hallo yuanhjty, great extension with a lot of potential.

I miss the snake-case-style with uppercase letters. 

For example:

```
//template.config.json

{
    "name": "Magento 1 Extension",
    "variables": ["extension_name"]
}
```
Value of extension_name = namespace_extension

Use in a template:
`___Extension_Name___`

Exspected result is "Namespace_Extension", but the result is "NamespaceExtension".
For building for Magento you need snake-case-style with uppercase letters. It would be great, if you can implement this.

NobyNa

**Comment by yuanhjty** [comment]
Hello @nobyna , `snake-case-style with uppercase letters` has been supported as `snakePascalCase` in newly released V0.3.0.

**Comment by nobyna** [comment]
Thank you. That 's working fine.
